src/interceptors/logging.interceptor.ts

```diff
@@ -80,20 +80,88 @@ export class LoggingInterceptor implements NestInterceptor {
             );
     }
 
-    filterResponse(response: unknown, outFields: AuditLogMetadata['outFields']) {
-        if (!outFields || !response || Object.getPrototypeOf(response) !== Object.prototype) {
+    filterResponse(response: unknown, outFields: AuditLogMetadata['outFields'], infields: AuditLogMetadata['infields']) {
+        let resp = response;
+
+        if (!outFields || !response || Object.getPrototypeOf(response) !== Object.prototype || infields === '*' || infields === 'all') {
             return {};
         }
 
+        if (infields && infields.length) {
+            for (let i = 0; i < infields.length; i++) {
+                delete resp[infields[i]];
```

Comment:
Since objects are assigned by reference, this would delete the properties from the response returned to the user. For example, adding `infields: ['orisCode'],` to the `exportMonitorPlan` AuditLog options will remove that field from the response JSON. I think, instead, those fields should just be skipped when building the result, as I noted in a different suggestion. Personally, I think this whole function could be combined with `filterRequestParams` (and renamed). Could remove the `boolean` from the `outFields` union type if it makes that easier.